vault/util_test.go

```diff
@@ -3,19 +3,40 @@
 
 package vault
 
-import "testing"
+import (
+	"math/rand"
+	"testing"
+)
+
+const n = 1024
 
 func TestMemZero(t *testing.T) {
-	b := []byte{1, 2, 3, 4}
+	r := rand.Intn(n)
```

Comment:
I don't think it matters but doesn't this have a 1/1024 chance of being zero-length?